openstack_controller/datadog_checks/openstack_controller/api.py

```diff
@@ -184,53 +186,13 @@ def get_projects(self, project_token):
         """
         Returns all projects in the domain
         """
-        # TODO: Is this call needed, we loop over all project and then call keystone for projects by giving
-        # the project token (not keystone)
+        # The project token (not keystone)
```

Comment:
I think this comment was chopped, it doesn't say anything at this point

Reply:
It's a note to myself but I should remove it. Thanks for the reminder